test/JDBC/sql_expected/BABEL-IDENTITY.out

```diff
@@ -423,24 +423,115 @@ int#!#int
 ~~END~~
 
 
--- Test get id max/min helper functions
-SELECT sys.get_min_id_from_table(';drop table master_dbo.test_table1;', 'master_dbo', 'test_table1');
-GO
+-- Test that identity counters shouldn't rolled back even if the transaction
+-- is rolled back.
+CREATE TABLE dbo.t1_identity_1(a int identity primary key, b int);
+SET IDENTITY_INSERT dbo.t1_identity_1 ON;
+INSERT INTO dbo.t1_identity_1 (a,b) VALUES (1,1);
+go
+~~ROW COUNT: 1~~
+
+
+-- Check transaction rollback should increase/decrease identity
+BEGIN TRAN t1; INSERT INTO dbo.t1_identity_1 (a,b) VALUES (300,2); ROLLBACK TRAN t1;
+SELECT @@IDENTITY; SELECT IDENT_CURRENT('dbo.t1_identity_1'); SELECT SCOPE_IDENTITY();
+go
+~~ROW COUNT: 1~~
+
+~~START~~
+numeric
+300
+~~END~~
+
 ~~START~~
-bigint
-~~ERROR (Code: 33557097)~~
+numeric
+300
+~~END~~
 
-~~ERROR (Message: column ";drop table master_dbo.test_table1;" does not exist)~~
+~~START~~
+numeric
+300
+~~END~~
 
 
-SELECT sys.get_max_id_from_table('test_id', ';drop table master_dbo', 'test_table1;');
-GO
+-- Check Statement error shouldn't increase/decrease identity
+BEGIN TRAN t1; INSERT INTO dbo.t1_identity_1 (a,b) VALUES (400,2); ROLLBACK TRAN t1;
```

Comment:
1. "should" increase/decrease identity?
2. It would be better to have an error case here instead of a rollback.